cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -483,7 +483,7 @@ object SnappyContextPerConnection {
     val context = connectionIdMap.get(connectionID)
     if (context != null) context
     else {
-      val snc = SnappyContext.getOrCreate(null: SparkContext).newSession()
+      val snc = SnappyContext(null: SparkContext).newSession()
```

Comment:
>>Yes, thats how it should be. Refer to SYS.SESSIONS table in GemXD. Otherwise all queries will get mixed in a single tab making it useless. 

Hmm. I agree that one cannot make out the queries fired from one connection if the tab is shared. But having multiple tabs is no solution either. See the picture on https://jira.snappydata.io/browse/SNAP-183. How can one easily figure out the SQL tab for the connection that he is looking for? I would still recommend postponing this PR and not change the UI behaviour for now. We can do it in a better way with 2.0, I suppose.